imports/lib/utils.js

```diff
@@ -90,6 +90,43 @@ export function removeTrailingSlash (str) {
   return str.replace(/\/$/, '')
 }
 
+export function setIsNavigatingBack (navigatingBack = false) {
+  Session.set('navigatingBack', navigatingBack)
+}
+
+export function getIsNavigatingBack () {
+  return Session.get('navigatingBack')
+}
+
+function initPreviousPaths () {
+  if (!Session.get('prevPaths')) {
```

Comment:
this is a bit of a minsnomer, because your are not "initializing" the prevPaths value, you are simply setting a default value if it is not defined. 

In any case, I think instead of doing

     initPReviousPaths()
     let x = Session.get('prevPaths')

It is cleaner to define a helper:

     function getNavHistory() {
          return Session.get('prevPaths') || []
    }

And use that wherever you now do `Session.get('prevPaths')`

Reply:
Another reason to do this is because now, for example, `getPreviousPath`, which you'd expect to just read a value, in some cases actually changes the Session as a side effect.

Reply:
Yeah, this was definitely a code smell so thanks for sniffing it out. I will refactor

Reply:
Changed via b01b743